web_custom_modifier/__manifest__.py

```diff
@@ -0,0 +1,19 @@
+# © 2023 Numigi (tm) and all its contributors (https://bit.ly/numigiens)
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl).
+
+{
+    "name": "Web Custom Modifier",
+    "version": "14.0.2.0.1",
```

Comment:
I think it should start from OCA version (not internal version)

Reply:
@francesco-ooops Right, it's our first try to push a module to OCA so, thank you for the review ;)

Reply:
> I think it should start from OCA version (not internal version)

No.

If the module exists before being shared into the OCA, revert to previous version could generate problem on odoo instances with the module installed.

Keep actual version is OK here.

Reply:
@legalsylvain another IMP for https://github.com/OCA/odoo-community.org/blob/master/website/Contribution/CONTRIBUTING.rst#id7 maybe? I think it could be worth opening a discussion on this.

FWIW, we use different names for internal modules (eg: ooops_web_custom_filter) and remove the ooops_ identifier at the beginning when publishing in OCA. Not only it makes it easier to distinguish them, but it's safer in case someone else on oca publishes a module with the same name.

anyway, @pedrobaeza what's your take on version?

Reply:
You don't need to start with `1.0.0` when you "move" a module to OCA. Another different story is when you migrate an existing OCA module, where the version is reset.

Reply:
> FWIW, we use different names for internal modules (eg: ooops_web_custom_filter) and remove the ooops_ identifier at the beginning when publishing in OCA. Not only it makes it easier to distinguish them, but it's safer in case someone else on oca publishes a module with the same name.

The prefix technique is valid to distinguish OCA / non OCA modules and avoid conflicts, indeed. That's the advantage.
But the drawbacks are numerous. If you installed "oops_module_xxx" modules, you have to handle migration when you want to switch to the OCA "module_xxx". in some cases, it can be boring. and this may prevent people from sharing their module in the OCA, which is not desirable.

Most of the contributors have so a "incubator" repositories, without prefix : 
- https://github.com/grap/grap-odoo-incubator
- https://github.com/akretion/ak-odoo-incubator
- https://github.com/acsone/acsone-addons
- https://github.com/Tecnativa/extra-addons
- https://github.com/camptocamp/c2c-rd-addons

...

Reply:
@legalsylvain thank you for sharing!

Reply:
To wrap it up: is perfectly fine. What is important is to mention this in the PR to not.... start such discussions :wink: